app/src/main/res/drawable/photo_camera_svgrepo_com.xml

```diff
@@ -0,0 +1,51 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"

```

Comment:
Maybe using a material icon to stay compliant with Material Design guidelines is a good idea?

Reply:
Yeah, this icon is not even used. It is the reminiscence of an old time, i'll fix it once at home

Reply:
Okay, good!